app/src/main/java/com/example/feelgoodinc/fragments/SelfHelpStrat.java

```diff
@@ -0,0 +1,66 @@
+package com.example.feelgoodinc.fragments;
+
+import android.os.Bundle;
+
+import androidx.fragment.app.Fragment;
+
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+
+import com.example.feelgoodinc.R;
+
+/**
+ * A simple {@link Fragment} subclass.
+ * Use the {@link SelfHelpStrat#newInstance} factory method to
+ * create an instance of this fragment.
+ */
+public class SelfHelpStrat extends Fragment {
+
+    // TODO: Rename parameter arguments, choose names that match
+    // the fragment initialization parameters, e.g. ARG_ITEM_NUMBER
+    private static final String ARG_PARAM1 = "param1";
+    private static final String ARG_PARAM2 = "param2";
+
+    // TODO: Rename and change types of parameters
+    private String mParam1;
+    private String mParam2;
+
+    public SelfHelpStrat() {
+        // Required empty public constructor
+    }
+
+    /**
+     * Use this factory method to create a new instance of
+     * this fragment using the provided parameters.
+     *
+     * @param param1 Parameter 1.
+     * @param param2 Parameter 2.
+     * @return A new instance of fragment self_help_strat.
+     */
+    // TODO: Rename and change types and number of parameters
+    public static SelfHelpStrat newInstance(String param1, String param2) {
+        SelfHelpStrat fragment = new SelfHelpStrat();
+        Bundle args = new Bundle();
+        args.putString(ARG_PARAM1, param1);
```

Comment:
Also not sure what `param1` and `param2` do here, probably auto-generated?

Should just remove them so we don't have any extra variables in the code.